lib/elixir/lib/string.ex

```diff
@@ -1439,4 +1439,51 @@ defmodule String do
   def to_float(string) do
     :erlang.binary_to_float(string)
   end
+
+  @doc """
+  Returns an integer representing Levenshtein distance between
+  `source` and `target`.
+
+  Levenshtein distance between two words is the minimum number of
+  single-character edits (i.e. insertions, deletions or substitutions)
+  required to change one word into the other
```

Comment:
In the docs. Search this module for "http://" and you will see a couple other places we link to outside.